### PR TITLE
docs: Adding Screenshot-Driven Development blog 

### PR DESCRIPTION
**Pull request by EbonyLouis**
## Reason
Adding @blackgirlbytes  `Screenshot-Driven Development` blog post 

## Changes
This PR includes:
- adding blog post `docs/blogs/posts/2024-11-22-screenshot-driven-development.md`
- adding blog images to `docs/blogs/images/screenshot-screenshot-driven-development-blog` folder
- creating blog category `Prototyping & Automation`

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208823712089503

### CODE REVIEW

File: docs/blog/posts/2024-11-22-screenshot-driven-development.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
+---
+draft: false
+title: "Screenshot-Driven Development"
+date: 2024-11-22
+authors:
+  - rizel
+categories:
+  - Prototyping & Automation
```
Comment (angiejones) [inline docs/blog/posts/2024-11-22-screenshot-driven-development.md]: how does this show up? as like one category? is it sort of like a tab. maybe need to add web development as a category

File: docs/blog/posts/2024-11-22-screenshot-driven-development.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+>Goose will prompt you to set up your API key when you first run this command. You can use various LLM providers like OpenAI or Anthropic
+
+```bash
+export OPENAI_API_KEY=your_api_key
+# Or for other providers:
+export ANTHROPIC_API_KEY=your_api_key
+```
+
+### Step 4: Enable the Screen Toolkit
```
Comment (angiejones) [inline docs/blog/posts/2024-11-22-screenshot-driven-development.md]:
```suggestion
### Step 4: Enable the Screen toolkit
```

File: docs/blog/posts/2024-11-22-screenshot-driven-development.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+Goose uses [toolkits](https://block.github.io/goose/plugins/plugins.html) to extend its capabilities. The [screen](https://block.github.io/goose/plugins/available-toolkits.html#6-screen-toolkit) toolkit lets Goose take and analyze screenshots.
+
+To enable the screen toolkit, add it to your Goose profile at ~/.config/goose/profiles.yaml.
```
Comment (angiejones) [inline docs/blog/posts/2024-11-22-screenshot-driven-development.md]:
```suggestion
To enable the Screen toolkit, add it to your Goose profile at ~/.config/goose/profiles.yaml.
```

File: docs/blog/posts/2024-11-22-screenshot-driven-development.md (hunk 1 above)
Comment (EbonyLouis) [reply docs/blog/posts/2024-11-22-screenshot-driven-development.md]: as we add more guides I figured we can add more categories 
how it looks in the blog tab:
Screenshot 2024-11-22 at 1 57 26 PM

how it looks in the sidebar when expanded:
Screenshot 2024-11-22 at 1 57 43 PM

Comment (angiejones) [reply docs/blog/posts/2024-11-22-screenshot-driven-development.md]: can we change it to Web Development?

Comment (EbonyLouis) [reply docs/blog/posts/2024-11-22-screenshot-driven-development.md]: yes

**Review by angiejones — APPROVED** [review]

**Pull request merged**